### PR TITLE
[QA-1759] Pragmatic fix for v1 billing project creation failure in tests

### PR DESCRIPTION
**Pull request by ehigham**
RR: https://broadworkbench.atlassian.net/browse/QA-1759
Bypass problematic deployment manager code for creating v1 billing
projects. Use GPAlloc instead. What could possibly go wrong?

The package path of the tests was also wrong and has been for quite a while. I'm not sure if this is intentional but the linter was giving me sad req squiggles so I fixed it + re-optimised imports.

**PR checklist**

- [ ] I've followed [the instructions](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've filled out the [Security Risk Assessment](https://broadinstitute.github.io/dsp-appsec-security-risk-assessment/) and attached the result to the JIRA ticket

### CODE REVIEW

**Review by gpolumbo-broad — APPROVED** [review]

File: automation/project/Dependencies.scala
Comment (andy7i) [inline automation/project/Dependencies.scala]: just checking -- will this be updated to an official version before merging?

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/sam/api/SamApiSpec.scala
Comment (andy7i) [inline automation/src/test/scala/org/broadinstitute/dsde/workbench/sam/api/SamApiSpec.scala]: the comment above mentions why we were using a brand new billing project intentionally. Since we've changed that now, we can probably remove the comment

Comment (andy7i) [reply automation/src/test/scala/org/broadinstitute/dsde/workbench/sam/api/SamApiSpec.scala]: since this change may cause other errors (which the comment alludes to), maybe we should leave a comment here as a breadcrumb in case it does cause issues?

On the other hand, I'm also ok with not leaving a comment since we can probably just look at the file history / git blame if/when we need to debug this test. 

On the other other hand, while leaving a comment would add some clutter, it would also save us time/effort in the future if we need to look at the test again and so I think I'm leaning toward adding a comment. 

Any thoughts?

**Review by andy7i — APPROVED** [review]
Approved after a couple updates -- probably should use an official version of wb-libs before merging this.

File: automation/project/Dependencies.scala
Comment (ehigham) [reply automation/project/Dependencies.scala]: Yep - waiting for https://github.com/broadinstitute/workbench-libs/pull/940

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/sam/api/SamApiSpec.scala
Comment (ehigham) [reply automation/src/test/scala/org/broadinstitute/dsde/workbench/sam/api/SamApiSpec.scala]: I'm not sure if this even passes yet! I'll remove the comment and update it when it passes!

**Pull request merged**
